tests/test_views.py

```diff
@@ -470,24 +470,25 @@ def test_cant_make_private_if_not_admin(self):
         assert_true(self.project.is_public)
 
     def test_add_tag(self):
-        url = "/api/v1/project/{0}/addtag/{tag}/".format(
+        url = "/api/v1/project/{0}/addtag/".format(
```

Comment:
Use `node.api_url_for()` for URL lookup here instead of hard-coded URLs: http://cosdev.readthedocs.org/en/latest/osf/guidelines.html#general